spikecomparison/studytools.py

```diff
@@ -127,30 +127,30 @@ def get_recordings(study_folder):
 
         rec = se.BinDatRecordingExtractor(raw_filename, info['sample_rate'], info['num_chan'],
                                           info['dtype'], frames_first=info['frames_first'])
-        se.load_probe_file(rec, prb_filename)
+        rec_probe = rec.load_probe_file(prb_filename)
 
-        recording_dict[rec_name] = rec
+        recording_dict[rec_name] = rec_probe
```

Comment:
Yep. Indeed. It was a bug. Good catch.
In the case, of GroundTrutghStudy the channel numbner is the same because the recording is saved in binary format the the good channel number.